### PR TITLE
[17.0][MIG] report_csv

### PR DESCRIPTION
**Pull request by pedroguirao**

### CODE REVIEW

File: report_csv/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "author": "Creu Blanca, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/reporting-engine",
     "category": "Reporting",
-    "version": "16.0.2.0.0",
+    "version": "17.0.2.0.0",
```
Comment (john-herholz-dt) [inline report_csv/__manifest__.py]: Should be 17.0.1.0.0, right?

**Review by john-herholz-dt — COMMENTED** [review]
Seems good except the version number.

File: report_csv/__manifest__.py (hunk 1 above)
Comment (pedroguirao) [reply report_csv/__manifest__.py]: Yes! thanks!

**Comment by ridge-works-fs** [comment]
Looks good and I was able to generate a CSV report against another model not already demonstrated. 

Are there other tests that I can assist with before this PR is merged?

**Review by john-herholz-dt — APPROVED** [review]

**Comment by yostashiro** [comment]
We have this refactoring PR https://github.com/OCA/reporting-engine/pull/864 for 16.0 which will affect this PR if accepted. Would be nice if we could get comments on the refactoring PR.

**Comment by bealdav** [comment]
Yes @yostashiro @pedroguirao 

Could you depend on this new module report_format_option please ? Thanks

https://github.com/OCA/reporting-engine/pull/889

**Pull request closed without merging**